libminifi/test/unit/FileUtilsTests.cpp

```diff
@@ -186,9 +184,8 @@ TEST_CASE("TestFileUtils::addFilesMatchingExtension", "[TestAddFilesMatchingExte
 
 TEST_CASE("FileUtils::last_write_time and last_write_time_point work", "[last_write_time][last_write_time_point]") {
   using namespace std::chrono;
-  namespace fs = std::filesystem;
 
-  fs::file_time_type time_before_write = file_clock::now();
+  file_clock::time_point time_before_write = file_clock::now();
```

Comment:
Just nitpicking, but it would be simpler to just use `auto` for these time points.

Reply:
you are absolutely right, I've fixed this in https://github.com/apache/nifi-minifi-cpp/pull/1615/commits/44c7d79b3d130f483271f92ef7b8b224533e4137